Source/EasyNetQ/Producer/Rpc.cs

```diff
@@ -229,11 +229,18 @@ protected virtual void RequestPublish<TRequest>(TRequest request, string routing
             var routingKey = conventions.RpcRoutingKeyNamingConvention(typeof(TRequest));
 
             var exchange = advancedBus.ExchangeDeclare(conventions.RpcExchangeNamingConvention(), ExchangeType.Direct);
-            var queue = advancedBus.QueueDeclare(routingKey);
+            var queue = advancedBus.QueueDeclare(routingKey, durable: configuration.Durable);
             advancedBus.Bind(exchange, queue, routingKey);
 
             return advancedBus.Consume<TRequest>(queue, (requestMessage, messageRecievedInfo) => ExecuteResponder(responder, requestMessage),
-                c => c.WithPrefetchCount(configuration.PrefetchCount));
+                c => c.WithPrefetchCount(configuration.PrefetchCount).WithRecoveryAction(() =>
+                    {
+                        //just in case we loose exchange and/or queue in case of owning cluster node went down, and they are nondurable
```

Comment:
Is this code executed even when the queue are set to be durable? If yes, it does not make sense to me because in a cluster a queue set to be durable cannot be re-declared in other node.